client/state/countries/reducer.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import { SMS, DOMAIN, PAYMENT } from './constants';
+import {
+	COUNTRIES_RECEIVE,
+	COUNTRIES_REQUEST,
+	COUNTRIES_REQUEST_FAILURE,
+	COUNTRIES_REQUEST_SUCCESS
+} from 'state/action-types';
+import { countriesSchema } from './schema';
+import { createReducer } from 'state/utils';
+
+export const items = createReducer( { [ SMS ]: [], [ DOMAIN ]: [], [ PAYMENT ]: [] }, {
```

Comment:
Do you suggest to also split up the event types per reducer like `COUNTRIES_SMS_RECEIVE` etc? If so, I feel like it's going to be a lot of bloat/boilerplate for the same stuff. We could use the same event internally, but I haven't seen encountered something like that either.